TYPO3.Flow/Classes/TYPO3/Flow/Utility/Files.php

```diff
@@ -377,7 +378,11 @@ public static function unlink($pathAndFilename)
         try {
             // if not on Windows, call PHPs own unlink() function
             if (DIRECTORY_SEPARATOR === '/' || is_file($pathAndFilename)) {
-                return @\unlink($pathAndFilename);
+                if (!@\unlink($pathAndFilename)) {
```

Comment:
It wouldn't be the same thing. Even after `file_exists` you still have the `unlink` fail, so… Even though `@` as evil as it gets, I think this is the only sane way.